components/sections/alphanet/Roadmap.tsx

```diff
@@ -18,19 +18,54 @@ function Roadmap() {
     {
       title: pageTranslation("page-alphanet-roadmap-1-title"),
       taskList: [
-        pageTranslation("page-alphanet-roadmap-1-tasklist-1"),
-        pageTranslation("page-alphanet-roadmap-1-tasklist-2"),
-        pageTranslation("page-alphanet-roadmap-1-tasklist-3"),
-        pageTranslation("page-alphanet-roadmap-1-tasklist-4"),
-        pageTranslation("page-alphanet-roadmap-1-tasklist-5"),
+        {
+          version: pageTranslation("page-alphanet-roadmap-1-version-1"),
+          tasks: [
+            pageTranslation("page-alphanet-roadmap-1-version-1-task-1"),
```

Comment:
can this be looped ? Is there a way to get the total number of elements in the translation json object and loop the keys ?

Reply:
ok, please add it in the github issues, so that it doesn't get missed.